src/main/webapp/script.js

```diff
@@ -82,3 +84,19 @@ function createButton(text) {
 function clearElm(elementID) {
     document.getElementById(elementID).innerHTML = "";
 }
+
+function userResult(){
+   
+    clearElm(QUIZ_ELEMENT_ID);
+    const quiz = document.getElementById(QUIZ_ELEMENT_ID);
+    quiz.appendChild(createParagraphElement("Based On Your Results You Should visit..."));
+   
+    getHeader(this);
+}
+
+function getHeader(){
+    var h = document.createElement("H3");
+    var t = document.createTextNode("Paris!");
```

Comment:
This variable is only used once, so it can be inlined into the next line:

`h.appendChild(document.createTextNode("Paris!"));`